packages/react-composites/src/composites/CallComposite/ScreenShare.tsx

```diff
@@ -84,8 +84,9 @@ export const ScreenShare = (props: ScreenShareProps): JSX.Element => {
 
     return (
       <VideoTile
-        displayName={screenShareParticipant?.displayName}
-        isMuted={screenShareParticipant?.isMuted}
+        displayName={
+          screenShareParticipant?.displayName ? `${screenShareParticipant?.displayName}'s screen` : undefined
```

Comment:
Maybe put it into the localization strings? Or just show the name should be good - no one will misunderstand a screensharing stream with a name to a real video stream